### DISCUSSION TITLE
PDF and Documents

### DISCUSSION THREAD

**Discussion by kalle07** [janhq/jan#1813] [Feature Ideas]
you planing support to talk with documents ?

**Comment by jmellinger473** [comment]
I'm with you, make it conversational,  an assistant... "hey Jan open mybills.xls." "Edit line 5 change billing amount to fifty dollars" or "Hey Jan whats the weather in New York today" etc....
PS I hate to type, I want it running on my system under my control so the trolls at google don't know!

**Comment by kalle07** [comment]
i mean at least like GPT4all   ... that you can aks for a consumtion of a long text 
its OPEN SOURCE you can COPY&pPaste   ;)

**Reply by TomLucidor** [reply]
That or Khoj but IDK how that can go.

**Comment by kalle07** [comment]
why is PDFGEAR  such as much better than all local llms with document reader iv tested ???
https://www.pdfgear.com/de/ 

300MB  local !!!
